docs/content/docs/checks/AndroidLint.md

```diff
@@ -45,3 +45,5 @@ How to start:
 
 - [Static Analysis with Android Lint by Tor Norbye (mDevCamp 2019)](https://slideslive.com/38916502) 
 - [Sample project](https://github.com/googlesamples/android-custom-lint-rules)
+- Install [PsiViewer IDE plugin](https://plugins.jetbrains.com/plugin/227-psiviewer/)
```

Comment:
Why did you add these links to block about Lint?

Reply:
- They contain useful information for writing custom lint checks.
- We don't have IDE plugins yet. They will have common things